packages/utils/src/libp2p/dialHelper.ts

```diff
@@ -167,13 +236,13 @@ async function doDial(
  * @param opts
  */
 export async function dial(
-  libp2p: LibP2P,
+  libp2p: ReducedLibp2p,
   destination: PeerId,
   protocol: string,
   opts?: DialOpts
 ): Promise<DialResponse> {
   return abortableTimeout(
-    (opts: TimeoutOpts) => doDial(libp2p, destination, protocol, opts as any),
+    (opts: Required<TimeoutOpts>) => doDial(libp2p, destination, protocol, opts),
```

Comment:
Does the lambda take `opts` from the `dial` argument as a closure, or from it's own argument? Probably should be renamed.

Reply:
Changed